src/lib/edge-functions/proxy.js

```diff
@@ -116,6 +116,7 @@ const initializeProxy = async ({
       [headers.ForwardedHost]: `localhost:${mainPort}`,
       [headers.Passthrough]: 'passthrough',
       [headers.RequestID]: generateUUID(),
+      [headers.IP]: '127.0.0.1',
```

Comment:
You can use the `LOCAL_HOST` constant.

```suggestion
      [headers.IP]: LOCAL_HOST,
```